### PR TITLE
less willing if more milscore

### PR DESCRIPTION
**Pull request by Nivaturimika**

### CODE REVIEW

File: src/ai/ai.cpp [hunk 1]
```diff
@@ -2111,7 +2111,7 @@ dcon::cb_type_id pick_fabrication_type(sys::state& state, dcon::nation_id from,
 
 bool valid_construction_target(sys::state& state, dcon::nation_id from, dcon::nation_id target) {
 	// Copied from commands.cpp:can_fabricate_cb()
-	if(from == target)
+	// if(from == target)
```
Comment (schombert) [inline src/ai/ai.cpp:2114]: uh, this function will now always return false ...

File: src/ai/ai.cpp [hunk 2]
```diff
@@ -3040,48 +3031,11 @@ void make_war_decs(sys::state& state) {
...
-
-			if(real_target == n)
-				continue;
-			if(nations::are_allied(state, n, real_target) || nations::are_allied(state, n, other))
-				continue;
-			if(real_target.get_in_sphere_of() == n)
-				continue;
-			if(state.world.nation_get_in_sphere_of(other) == n)
-				continue;
-			if(military::has_truce_with(state, n, other) || military::has_truce_with(state, n, real_target))
```
Comment (schombert) [inline src/ai/ai.cpp:3057]: why did all these checks get removed?

File: src/ai/ai.cpp (hunk 1 above)
Comment (Nivaturimika) [reply src/ai/ai.cpp]: FUCK HOW AM I SO IDIOTIC

File: src/ai/ai.cpp (hunk 2 above)
Comment (Nivaturimika) [reply src/ai/ai.cpp]: fixed

**Pull request merged**